internal/service/cloudbackupschedule/resource_cloud_backup_schedule_test.go

```diff
@@ -46,6 +47,7 @@ func TestAccBackupRSCloudBackupSchedule_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "policy_item_monthly.#", "0"),
 					resource.TestCheckResourceAttr(resourceName, "policy_item_yearly.#", "0"),
 					resource.TestCheckResourceAttr(dataSourceName, "cluster_name", clusterInfo.Name),
+					resource.TestCheckResourceAttr(dataSourceName, "cluster_name", clusterInfo.Name),
```

Comment:
small nit: will remove in followup PR or if changes are required as part of this PR (to avoid retriggering tests)